### PR TITLE
added  data model

### LINKED ISSUE #12: ft-Data storage, Models and CRUD processes for users, fund and project management
**Is your feature request related to a problem? Please describe.**
The platform is data driven and thus requires a database for data storage and accessibility

**Describe the solution you'd like**
Cloud hosted  MySql database, Data Models and CRUD processes to manage data transactions for:
- Users Management
-Fund Management
-Project Management

**Describe alternatives you've considered**
local hosting of MySql database and writing raw Sql queries in the application

### PR DESCRIPTION
**Pull request by victoryohanna**
## Description
Data model is created base on our data schema using Sequelize ORM as earlier agreed.
Controllers and routes are created to test DB connection and CRUD operation carried out for few model on localhost before remote database is setup. 
 
## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ x] I have added tests that prove my fix is effective and that this feature works
- [x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by emydame — COMMENTED** [review]
Hi Victor,

Kindly confirm where the database connection is pointing to.

Thanks

**Comment by Onyedikachi** [comment]
Hi Victor, the recent update to the develop branch produced this conflict. Kindly merge develop branch into the backend-model branch.

Resolve the conflicts and re-push the backend-model branch

File: src/backend/model/user.model.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module.exports = (sequelize, Sequelize) => {
```
Comment (Onyedikachi) [inline src/backend/model/user.model.js:1]: Hi victor user model and user categories model are the same

File: src/backend/yarn.lock [hunk 2]
```diff
@@ -8268,11 +8293,14 @@ wrap-ansi@^6.2.0:
     string-width "^4.1.0"
     strip-ansi "^6.0.0"
 
+>>>>>>> 91ba67f90d23f917d22b768ffb92953ed45b7c4e
 wrappy@1:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/wrappy/-/wrappy-1.0.2.tgz#b5243d8f3ec1aa35f1364605bc0d1036e30ab69f"
   integrity sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=
 
+<<<<<<< HEAD
```
Comment (Onyedikachi) [inline src/backend/yarn.lock]: Pls address this line

File: src/backend/yarn.lock [hunk 3]
```diff
@@ -8079,11 +8079,21 @@ utils-merge@1.0.1:
   resolved "https://registry.yarnpkg.com/utils-merge/-/utils-merge-1.0.1.tgz#9f95710f50a267947b2ccc124741c1028427e713"
   integrity sha1-n5VxD1CiZ5R7LMwSR0HBAoQn5xM=
 
+<<<<<<< HEAD
```
Comment (Onyedikachi) [inline src/backend/yarn.lock]: please address this whole file, you can recreate this file

**Review by Onyedikachi — CHANGES_REQUESTED** [review]

**Review by victoryohanna — COMMENTED** [review]
Issues resolved

**Review by Onyedikachi — APPROVED** [review]

**Comment by Onyedikachi** [comment]
Approved

**Comment by Onyedikachi** [comment]
Hi Victor, the latest merge to develop produced a new conflict, Kindly resolve

File: src/backend/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const app = express();
+const PORT = process.env.PORT || 3000;
```
Comment (MosesNwaeze) [inline src/backend/index.js:4]: this port will conflict with the frontend port that runs on 3000

**Review by MosesNwaeze — APPROVED** [review]

**Review by emydame — APPROVED** [review]
Reviewed

**Comment by emydame** [comment]
Hi Victor, 
Kindly resolve the conflict issue in the yarn file contained in the back-end directory to enable us merge the pull request. It is a versioning conflict and i am not certain which version you might be needing for the backend.

Thank you.

**Review by elfizzy — APPROVED** [review]
Reviewed

**Comment by emydame** [comment]
This request is closed. Resolution of the conflict shall be handled in another pull request

**Pull request closed without merging**